src/api/app/models/project.rb

```diff
@@ -135,16 +139,8 @@ def cleanup_before_destroy
       end
     end
 
-    # revoke all requests
-    # FIXME: this is breaking request accepting with cleanup, if the project
-    #        will be removed during that
-#    revoke_requests
-  end
-
-  def delete_on_backend(comment = nil)
-    path = source_path
-    path << Suse::Backend.build_query_from_hash({user: User.current.login, comment: comment}, [:user, :comment])
-    Suse::Backend.delete path
+    # revoke all requests that have this project as source/target
+    revoke_requests
```

Comment:
On Dienstag, 6. Oktober 2015, 08:34:39 CEST wrote Bj�rn Geuken:
> > @@ -135,16 +139,8 @@ def cleanup_before_destroy
> >        end
> >      end
> >  
> > -    # revoke all requests
> > -    # FIXME: this is breaking request accepting with cleanup, if the project
> > -    #        will be removed during that
> > -#    revoke_requests
> > -  end
> > -
> > -  def delete_on_backend(comment = nil)
> > -    path = source_path
> > -    path << Suse::Backend.build_query_from_hash({user: User.current.login, comment: comment}, [:user, :comment])
> > -    Suse::Backend.delete path
> > +    # revoke all requests that have this project as source/target
> > +    revoke_requests
> 
> Cool
> 

revoking when it is used as target sounds wrong, shouldn't it be declined then?

-- 

Adrian Schroeter
email: adrian@suse.de

SUSE Linux GmbH, GF: Felix Imend�rffer, Jane Smithard, Graham Norton, HRB 21284 (AG N�rnberg) 
 
Maxfeldstra�e 5                         
90409 N�rnberg 
Germany